test/integration/taxonomy-integration.ts

```diff
@@ -0,0 +1,404 @@
+import { getTestOrganizationId, initPlainClient } from '../helpers'
+import { afterEach, beforeEach, describe, test } from 'mocha'
+import { expect } from 'chai'
+import { ConceptProps, CreateConceptProps } from '../../lib/entities/concept'
+import { ConceptSchemeProps, CreateConceptSchemeProps } from '../../lib/export-types'
+
+let conceptsToDelete: ConceptProps[] = []
+let conceptSchemesToDelete: ConceptSchemeProps[] = []
+
+describe('Taxonomy Integration', () => {
+  const client = initPlainClient({
+    organizationId: getTestOrganizationId(),
+  })
+
+  beforeEach(() => {
+    conceptsToDelete = []
+    conceptSchemesToDelete = []
+  })
+
+  afterEach(async () => {
+    for (const concept of conceptsToDelete) {
```

Comment:
More of a general question: if/when this fails to trigger (e.g. someone kills the build half way through), what is the impact? is it just debris left in the space or could it cause flakey tests in subsequent test runs?

Reply:
Also I'm not quite clued up on the test setup: but would it be easier and cleaner to wipe the whole space before/after a test run?

Reply:
Maybe you're right, I'm not sure about the thought behind it either.